tests/normalizers/test_ftp.py

```diff
@@ -0,0 +1,285 @@
+"""Tests for the FTP normalizer of some ftp resources"""
+import unittest
+from collections import OrderedDict
+from datetime import datetime
+
+from dateutil.tz import tzutc
+
+import metanorm.normalizers as normalizers
+
+
+class FTPMetadataNormalizerTestCase(unittest.TestCase):
+    """Tests for the FTP normalizer"""
+    @classmethod
+    def setUpClass(cls):
+        cls.normalizer = normalizers.FTPMetadataNormalizer([], [])
+
+    def test_time_coverage_start_remss_single_day_file(self):
+        """shall return the propert starting time for hardcoded normalizer """
+        self.assertEqual(
+            self.normalizer.get_time_coverage_start(
+                {'ftp_domain_name': 'ftp.remss.com',
+                 'ftp_add_and_file_name': 'gmi/bmaps_v08.2/y2014/m06/f35_201406v8.2.gz'}),
```

Comment:
```suggestion
                {'url': 'ftp://ftp.remss.com/gmi/bmaps_v08.2/y2014/m06/f35_201406v8.2.gz'}),
```

Please fix also in all tests below.

Reply:
Done in the next commit